### ISSUE TITLE
[feature-req] 

### ISSUE THREAD

**Issue by davidkarlsen** [jOOQ/jOOQ#7665]
Currently jooq supports change-timestamp ootb.
We are currently on JPA, which supports both an inserted and updated timestamp which is useful as you know when the record was initially created and when it was last updated - a feature we use.
Could you consider implementing created timestamp in jooq as well?

**Comment by lukaseder** [comment]
Thanks for your feature request. I'm assuming you mean this to be triggered on `UpdatableRecord.store()`, `insert()`, and `update()` calls, right? As an extension to the current optimistic locking timestamp...

In that case, this would be the same feature request as : https://github.com/jOOQ/jOOQ/issues/1592.

Note, you can already work around the missing functionality by implementing a `RecordListener`, or, of course, a database trigger, which would be a more thorough solution.

**Comment by davidkarlsen** [comment]
Yes, it looks like the same - and I see I actually commented back 3 years ago - hard to remember :) So I guess it won't be implemented ootb. I am aware that I can roll my own, but want to get most out of the potential investment OOTB.
